sdks/python/apache_beam/runners/interactive/sql/utils.py

```diff
@@ -161,7 +161,8 @@ class OptionsForm:
   generate PipelineOptions to run pipelines.
   """
   def __init__(self):
-    self.options = PipelineOptions()
+    # ignore all flags
```

Comment:
Could you add a link to a TODO bug, or an expanded comment explaining why this is needed here?

Reply:
I added more comments to explain why we propose this fix.